virttest/utils_time.py

```diff
@@ -0,0 +1,229 @@
+import logging
+import re
+
+from avocado.utils import process
+
+from virttest import error_context
+
+
+def get_host_timezone(params):
+    """
+    Timedatectl is installed with systemd.
+    :param params: Dictionary with the test parameters
+
+    :return: timez_city: the host's timezone in city format, used in Linux os.
+    :return: timez_code: the hosts timezone in utc code format, used in both
+                         Linux and Win.
+    """
+    try:
+        o = process.system_output(params["check_host_tz_cmd"], timeout=240,
+                                  shell=True)
+    except process.CmdError:
+        logging.warn("Can not get host's timezone")
+    tz = re.match(r'^(?:\s+Time zone:\s)(\w+\/\w+|UTC)(?:\s\(\w+,\s)([+|-]'
+                  r'\d{4})\)$', o, re.M | re.I)
+    if tz:
+        timez_city = tz.groups()[0]
+        timez_code = tz.groups()[1]
+    else:
+        timez_city = ""
```

Comment:
Better put these two lines, before `try`
`timez_city = ""`
 `timez_code = ""`
Then, you don't need this `else`.

Reply:
Agree with @hereischen

Reply:
Thanks! Update it.